src/ec.ml

```diff
@@ -489,19 +489,32 @@ let main () =
                    let loc = p.EP.gl_action.EcLocation.pl_loc in
                    let timed = p.EP.gl_debug = Some `Timed in
                    let break = p.EP.gl_debug = Some `Break in
+                   let ignore_fail = ref false in
                      try
-                       let tdelta =
-                         EcCommands.process ~timed ~break p.EP.gl_action
-                       in tstats loc tdelta
+                       let tdelta = EcCommands.process ~timed ~break p.EP.gl_action in
+                       if p.EP.gl_fail then begin
+                         ignore_fail := true;
+                         raise (EcScope.HiScopeError (None, "this command is expected to fail"))
```

Comment:
For convenience in interactive mode, could the location of this error message be narrowed down to the command that does not fail rather than the `fail` keyword itself?

Reply:
Oh, and feel free to resolve this if you choose to ignore it.

Reply:
Currently, the error spans the full command (`fail` included).  I don't have the position information for the command without "fail".

<img width="868" alt="image" src="https://github.com/EasyCrypt/easycrypt/assets/1540508/af9e8ce4-c4c4-4b2e-9275-aff10fb819f2">

Reply:
Yes. I think it's fine to keep as is, especially given this is not being introduced to help with interactive proofs.